app/src/main/java/com/example/teamyuml/decentworkmobile/views/NoticeList.java

```diff
@@ -1,144 +1,88 @@
-                        }
-
-                        if (noticeAll != null) {
-                            initNoticeList();
-                            Toast.makeText(NoticeList.this,
-                                    "Pobrano " + noticeAll.size() + " ogłoszenia",
-                                    Toast.LENGTH_LONG).show();
-                        }
-                    }
-
-                }, new Response.ErrorListener() {
-                    @Override
-                    public void onErrorResponse(VolleyError error) {
-                        Toast.makeText(NoticeList.this,
-                                "Coś poszło nie tak",
-                                Toast.LENGTH_LONG).show();
-                    }
-                });
-
-        VolleyInstance.getInstance(this).addToRequestQueue(jsonObjectRequest, "noticeList");
-    }
-
-    /**
-     * Set adapter to ListView and add clickListener event to get clicked data
-     */
-    private void initNoticeList() {
-        CustomListView adapter = new CustomListView(this, noticeAll);
-        noticeList.setAdapter(adapter);
-
-        noticeList.setOnItemClickListener(new AdapterView.OnItemClickListener() {
-            public void onItemClick(AdapterView<?> parent, View v, int position, long id) {
-                String clickedItem = noticeAll.get(position).get("id").toString();
-                Intent toNoticeDatail = new Intent(NoticeList.this, NoticeDetails.class);
-                toNoticeDatail.putExtra("choosenNotice", (String) clickedItem);
-                startActivity(toNoticeDatail);
-
-            }
-        });
-    }
-
     private void panelSpinnerAdapter() {
         ArrayAdapter<CharSequence> spinnerAdapter = ArrayAdapter.createFromResource(
-                this, R.array.notice_list, android.R.layout.simple_spinner_dropdown_item);
+            this, R.array.notice_list, android.R.layout.simple_spinner_dropdown_item);
         spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
         panelSpinner.setAdapter(spinnerAdapter);
 
         panelSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
             @Override
             public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
-                if (parent.getItemIdAtPosition(position) == 1) {
-                    Intent toWorker = new Intent(NoticeList.this ,Worker.class);
-                    startActivity(toWorker);
+                FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+
+                if (parent.getItemIdAtPosition(position) == 0) {
+                    Fragment notice = new ListViewFragment();
+
+                    notice.setArguments(setParameters(
+                        NOTICE_URL,
+                        R.id.noticeList,
+                        R.layout.notice_list_view,
+                        "getNotice",
+                        "noticesList"
```

Comment:
These two names noticesList and workersList are wrong. If we want call class by name we should use class name like "WorkerDetails"